sysutils/netdata/Portfile

```diff
@@ -67,25 +65,66 @@ set netdata_web_dir     ${prefix}/share/${name}/web
 
 add_users ${netdata_user} group=${netdata_group}
 
-use_autoreconf yes
+if { ${name} eq ${subport} } {
+    use_autoreconf yes
+    depends_run-append      port:${name}-dashboard
+
+    startupitem.create      yes
+    startupitem.executable  ${prefix}/sbin/netdata -D
+
+    destroot.keepdirs-append \
+                            ${destroot}${netdata_cache_dir} \
+                            ${destroot}${netdata_log_dir} \
+                            ${destroot}${netdata_lib_dir}
+
+    post-destroot {
+        xinstall -m 0644 ${worksrcpath}/system/netdata.conf ${destroot}${netdata_conf_dir}
+
+        reinplace "s|web files owner = .*|web files owner = netdata|" ${destroot}${netdata_conf_dir}/netdata.conf
+        reinplace "s|NETDATA_USER_CONFIG_DIR=\"/|NETDATA_USER_CONFIG_DIR=\"${prefix}/|" ${destroot}${netdata_conf_dir}/edit-config
+        reinplace "s|NETDATA_STOCK_CONFIG_DIR=\"/|NETDATA_STOCK_CONFIG_DIR=\"${prefix}/|" ${destroot}${netdata_conf_dir}/edit-config
+
+        xinstall -d -o ${netdata_user} -g ${netdata_group} ${destroot}${netdata_cache_dir}
+        xinstall -d -o ${netdata_user} -g ${netdata_group} ${destroot}${netdata_log_dir}
+        xinstall -d -o ${netdata_user} -g ${netdata_group} ${destroot}${netdata_lib_dir}
+
+        touch ${destroot}${netdata_conf_dir}/.opt-out-from-anonymous-statistics
+
+        # Delete any files provided by netdata-dashboard to avoid conflicts. We
+        # can't just delete the whole directory because this directory contains
+        # some files that are not present in netdata/dashboard. This will
+        # hopefully change someday, see
+        # https://github.com/netdata/dashboard/issues/83
+        foreach f [glob -dir ${netdata_web_dir} * **/*] {
+            delete ${destroot}${f}
+        }
```

Comment:
@cjones051073 In particular, take a look at this block. This is a bit of a tricky situation, as `netdata/netdata` provides some of the static web files that are *not* present in `netdata/dashboard`, so in order to prevent conflicts upon installation we have to delete the files provided by `netdata/dashboard` while preserving the ones from `netdata/netdata`. If you know of a better way to do this I'm all ears.